arm-network/2016-06-01/swagger/network.json

```diff
@@ -7234,6 +7234,49 @@
       ],
       "description": "IPConfiguration"
     },
+    "ResourceNavigationLinkFormat": {
+      "properties": {
+        "linkedResourceType": {
+          "type": "string",
+          "description": "Gets or sets the resource type of the linked resource"
+        },
+        "link": {
+          "type": "string",
+          "description": "Gets or sets the link to the external resource"
+        },
+        "parentResourceGuid": {
+          "type": "string",
```

Comment:
parentResourceGuid presence was a mistake. I removed it entirely. Will regenerate the SDK as well.